scripts/run_unit_tests.sh

```diff
@@ -11,6 +11,8 @@ cargo_args=("")
 if [[ $hypervisor = "mshv" ]]; then
     cargo_args+=("--no-default-features")
     cargo_args+=("--features common,$hypervisor")
+else
+    cargo_args+=("--features tdx")
```

Comment:
Looks like we should only enable `tdx` feature for `x86_64` build.

Reply:
Yes you're right! Done!